src/css/components/_modal-dialog.scss

```diff
@@ -0,0 +1,33 @@
+.video-js .vjs-modal-dialog {
+  @extend %fill-parent;
+  @include linear-gradient(180deg, rgba(0, 0, 0, 0.8), rgba(255, 255, 255, 0));
+}
+
+.vjs-modal-dialog .vjs-modal-dialog-content {
+  @extend %fill-parent;
+  z-index: 1;
+}
+
+// Ultimately, this would be moved to its own component partial, but
+// for now it's unclear what styles would be generic and which specific
+// to this context!
+.vjs-modal-dialog .vjs-close {
```

Comment:
can we move the close button styles into its own thing? Also, inherit from the close-button font icon thing as appropriate?